src/Microsoft.DotNet.Configurer/NuGetCacheSentinel.cs

```diff
@@ -23,7 +23,7 @@ private string NuGetCachePath
             {
                 if (string.IsNullOrEmpty(_nugetCachePath))
                 {
-                    _nugetCachePath = PackageDependencyProvider.ResolvePackagesPath(null, null);
+                    _nugetCachePath = NuGetPathContext.Create(new NullSettings()).UserPackageFolder;
```

Comment:
Using NullSettings NUGET_PACKAGES will be used. But NuGet.Config files in the project folder or solution folder will not be used, machine wide settings will also be ignored.

Is there anyway you can get the project.json directory here to load settings from that location? How is this class used?